ci_images/dnf_wrapper.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/sh
```

Comment:
Does this need `set -euxo pipefail`?

Reply:
Added everything but `x`. Since this command will run in CI workloads, we don't want it to be very verbose and muddy up the logs generated by CI builds.